src/hivemq-extension/conf/examples/config.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<extension-configuration>
```

Comment:
Should we remove the example `mqttMessageLog.properties` from the package?
This encourages customers to use `config.xml` but may feel forceful to customers used to the legacy config.
If we leave it in the package but customers set `config.xml`, the xml configuration is used.

Also, `<extension-configuration>` here is consistent with File RBAC extension but I'd rather change it to `<hivemq-mqtt-message-log-extension>`

Reply:
Yes I would also remove the `mqttMessageLog.properties` example file. 
+1 on changing the root tag to `hivemq-mqtt-message-log-extension`